draft-ietf-quic-recovery.md

```diff
@@ -228,13 +228,17 @@ more accurate round-trip time estimate; see Section 13.2 of {{QUIC-TRANSPORT}}.
 ## Probe Timeout Replaces RTO and TLP
 
 QUIC uses a probe timeout (PTO; see {{pto}}), with a timer based on TCP's RTO
-computation.  QUIC's PTO includes the peer's maximum expected acknowledgment
-delay instead of using a fixed minimum timeout. QUIC does not collapse the
-congestion window until persistent congestion ({{persistent-congestion}}) is
-declared, unlike TCP, which collapses the congestion window upon expiry of an
-RTO.  Instead of collapsing the congestion window and declaring everything
-in-flight lost, QUIC allows probe packets to temporarily exceed the congestion
-window whenever the timer expires.
+computation; see {{?RFC6297}}.  QUIC's PTO includes the peer's maximum expected
+acknowledgment delay instead of using a fixed minimum timeout.
+
+As reflected by TLP defined in RACK ({{?RACK=I-D.ietf-tcpm-rack}}), a single
+packet loss at the tail does not indicate persistent congestion, so QUIC does
+not collapse the congestion window when the PTO expires.  Instead of
+collapsing the congestion window and declaring everything in-flight lost, QUIC
+specifies a time-based definition to ensure one or more packets are sent prior
+to a dramatic decrease in congestion window; see {{persistent-congestion}}.
+QUIC allows probe packets to temporarily exceed the congestion window
+whenever the timer expires.
```

Comment:
```suggestion
Similar to the RACK-TLP loss detection algorithm for TCP
({{?RACK=I-D.ietf-tcpm-rack}}), QUIC does not collapse the congestion window
when the PTO expires, since a single packet loss at the tail does not indicate
persistent congestion.  Instead, QUIC collapses the congestion window when
persistent congestion is declared; see {{persistent-congestion}}.

QUIC allows probe packets to temporarily exceed the congestion window whenever
the timer expires.
```